src/host/ft_host/precomp.h

```diff
@@ -3,6 +3,8 @@
 
 #pragma once
 
+#define NOMINMAX

```

Comment:
Should I add some comments here? I assume everybodys knows what it is.

Reply:
I think it's fair to leave as-is, it's everywhere now!